src/schema/v2/artist/verifiedRepresentatives.ts

```diff
@@ -0,0 +1,51 @@
+import { GraphQLFieldConfig, GraphQLList, GraphQLNonNull } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { PartnerType } from "schema/v2/partner/partner"
+
+const VerifiedRepresentatives: GraphQLFieldConfig<
+  { _id: string },
+  ResolverContext
+> = {
+  type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(PartnerType))),
+  resolve: async (
+    { _id },
+    options,
+    { verifiedRepresentativesLoader, partnersLoader }
+  ) => {
+    if (!verifiedRepresentativesLoader) {
```

Comment:
You shouldn't need this, as this is declared in the 'without' authentication factory, it will always be available. There's no authorization possible (as `X-Access-Token` header won't be propagated).